react-carousel-component/src/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Carousel from './carousel';
+
+const characters = [
+  {
+    greninja: 'https://www.smashbros.com/assets_v2/img/fighter/greninja/main8.png'
+  },
+  {
+    mewtwo: 'https://www.smashbros.com/assets_v2/img/fighter/mewtwo/main8.png'
+  },
+  {
+    fox: 'https://www.smashbros.com/assets_v2/img/fighter/fox/main.png'
+  },
+  {
+    samus: 'https://www.smashbros.com/assets_v2/img/fighter/samus/main8.png'
+  }
```

Comment:
It's better to make a homogenous data model where the key names and types are the same. I recommend a `name` key and an `imageUrl` key for each object.